### ISSUE TITLE
[K8s][Tests] Add API to reset the network state for E2E tests

### ISSUE THREAD

**Issue by okdas** [pokt-network/poktroll#673]
## Objective

We need a way to reset the network to a particular genesis app state to run specific tests (e.g. there should be a staked actor to unstake it).

## Origin Document

https://www.notion.so/K8sNet-0bc00f33aa4549e7acf4e70be78c6cfc?pvs=4

## Goals

- Expose an API to reset the network.
- Ideally, this API should work on both DevNet and LocalNet.
- We need e2e tests isolated. Only after that work is done, we can have an isolated e2e tests.

## Deliverables

- [ ] A testutil package that restarts the validator with a desired state.

## Non-goals / Non-deliverables

- Switching tests to actually use this package. That work comes later.

## General deliverables

- [ ] **Comments**: Add/update TODOs and comments alongside the source code so it is easier to follow.
- [ ] **Testing**: Add new tests (unit and/or E2E) to the test suite.
- [ ] **Makefile**: Add new targets to the Makefile to make the new functionality easier to use.
- [ ] **Documentation**: Update architectural or development READMEs; use [mermaid](https://mermaid-js.github.io/mermaid/) diagrams where appropriate.

---

**Creator**: @okdas

**Comment by Olshansk** [comment]
Too broad of an idea/task. Closing it out.
